euctr/templates/about.html

```diff
@@ -232,6 +232,20 @@ <h2 id="part-of">What Happens when Sponsors Merge or Split?</h2>
         acquisitions; if you think we should update any of our data on
         this, please <a href="#contact">tell us</a>.
         </p>
+        
+        <h2 id="part-of">What is the impact of Brexit?</h2>
```

Comment:
the `id` in the `<h2>` is to allow others to link directly to this section of the page & should therefore be unique on the page. e.g. https://eu.trialstracker.net/about#major-sponsor , https://eu.trialstracker.net/about#inconsistent-data

Anyway, this unique id `part-of` already occurs three times! We can fix the others another time, but how about we do the right thing in this PR & make this something like:

`<h2 id="brexit">What is the impact of Brexit?</h2>`

Reply:
Ohhh, I'm a complete HTML newbie as far as what the functionality of anything is, so this is just undoubtedly just me copy/pasting blinding without regard for this! I've gone through and adjusted throughout with this in mind!

Reply:
ah cool, thanks for fixing!